xpdacq/utils.py

```diff
@@ -42,8 +42,8 @@ def composition_analysis(compstring):
     fractions = [getfraction(w) for w in namefracs[1::2]]
     return names, fractions
 
-def export_usermetadata():
-    """ functions exporting user meata stored in config_base and userScript under xpdUser directory
+def export_userScript_etc():
```

Comment:
Please change the name to ``export_userScriptsEtc()``

too many underscores...